src/middlewared/middlewared/plugins/audit/audit.py

```diff
@@ -193,9 +197,27 @@ async def query(self, data):
         `success` - boolean value indicating whether the action generating the
         event message succeeded.
         """
-        sql_filters = data['query-options']['force_sql_filters']
 
         verrors = ValidationErrors()
+
+        # If HA, handle the possibility of remote controller requests
+        ctrlr_state = await self.middleware.call('failover.status')
```

Comment:
There's no reason to query `failover.status`. You can call `if failover.licensed and data['remote_controller']` and then just try to download the remote logs since you're covering the failure conditions down below.